lib/main.ts

```diff
@@ -497,19 +499,30 @@ export class Transpiler {
     this.emit(translated);
   }
 
-  static DART_KEYWORDS =
-      ('abstract as assert async await break case catch class const continue ' +
-       'default deferred do dynamic else enum export extends external factory false final ' +
-       'finally for get if implements import in is library new null operator part rethrow return' +
-       ' set static super switch sync this throw true try typedef var void while with yield')
-          .split(/ /);
+  // Dart keyword list copied from
+  // https://www.dartlang.org/docs/dart-up-and-running/ch02.html#keywords
+  // Note the different kinds of keywords defined by the enum; key values
+  // match the footnote numbers given in the docs, or 0 if none given.
+  static DART_KEYWORDS : { [key:string] : DART_KEYWORD_KIND } = {
```

Comment:
IMHO, a proper solution to #113 would be to:
* throw an error if an identifier is a RESERVED keyword and
* issue a warning otherwise.
Dart docs say that BUILTIN ids should be avoided but are ok to use.
Hence the need for the distinction in keywords.